sys/Makefile.dep

```diff
@@ -109,6 +109,9 @@ ifneq (,$(filter gnrc_gomach,$(USEMODULE)))
 endif
 
 ifneq (,$(filter gnrc_lorawan,$(USEMODULE)))
+  USEMODULE += ztimer
```

Comment:
`ztimer` is already a dependency of `ztimer_msec`:

https://github.com/RIOT-OS/RIOT/blob/8fcafb89d05c481b56af85244914d41fa4038475/sys/ztimer/Makefile.dep#L82-L84

Reply:
done! :) (https://github.com/RIOT-OS/RIOT/pull/14558/commits/24923e80a1dbf088c78b497ad21b91fefb3551d9)